### DISCUSSION TITLE
Pruning old after update using dockge - how to?

### DISCUSSION THREAD

**Discussion by randall11GB** [louislam/dockge#587] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

after updating an image, i want to remove the old unused. can this be done in dockge?

### 🐻 Dockge Version

1.42

### 💻 Operating System and Arch

debian 12

### 🌐 Browser

thorium

### 🐋 Docker Version

24.0.7

**Comment by cyril59310** [comment]
open the console 
![image](https://github.com/user-attachments/assets/fd0d8535-cb5f-4894-ab9b-ac4994e75048)

listed images not used `docker images -f dangling=true`

![image](https://github.com/user-attachments/assets/eae463f9-f2a2-42c0-890f-f4e6e2687374)

and use `docker rmi ImageID` for delete specifique images 

or
use `docker image prune` for delete all images not used

**Comment by randall11GB** [comment]
Worked perfectly! Thank you for this.

-------- Original Message --------

**Comment by IetIesAai** [comment]
Woud be nice to be able to do that just from the interface though....

**Comment by trip5** [comment]
+1 for doing it from the interface...
